.pylintrc

```diff
@@ -102,7 +102,7 @@ ignore-patterns=
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().
-#init-hook=
+init-hook='import sys; sys.path.append("autoscheduler")

```

Comment:
Of course now I can't figure out how to replicate the issue, so try to remove this line and run pylint in `scraper/`. I believe it was causing an error for `from scraper.banner_requests import Semester, ...` in `banner_requests_tests.py`, but now its not throwing one for me.

Reply:
Scratch that, the actual issue was trying to import `from scraper.banner_requests import BannerRequests` in `scraper_courses.py`, which I succeeded in replicating. so I do believe this is necessary.